examples/lighting-app/qpg/BUILD.gn

```diff
@@ -44,9 +44,7 @@ qpg_sdk("sdk") {
     "${qpg_project_dir}/include",
   ]
 
-  if (chip_enable_pw_rpc) {
-    defines = [ "PW_RPC_ENABLED" ]
-  }
+  defines = [ "PW_RPC_ENABLED=${chip_enable_pw_rpc}" ]
```

Comment:
I guess the semantics of this macro (boolean value vs just-defined-or-not) are per-platform, so it's OK that this is not necessarily going to be consistent with other platforms?